pkg/datapath/linux/modules/modules.go

```diff
@@ -30,8 +36,12 @@ func (m *Manager) FindModules(expectedNames ...string) (bool, []string) {
 }
 
 // FindOrLoadModules checks whether the given kernel modules are loaded and
-// tries to load those which are not.
-func (m *Manager) FindOrLoadModules(expectedNames ...string) error {
+// tries to load those which are not. Skip this validation if instructed to do so.
+func (m *Manager) FindOrLoadModules(skipValidation bool, expectedNames ...string) error {
+	if skipValidation {
+		log.Warningf("skipped validating and loading the following kernel modules: %s)", expectedNames)
```

Comment:
I disagree that this is a Warning. It's a configuration choice, make it an Info log so that future us can see it when debugging. If we're worried that it breaks too often then I guess we shouldn't make it an option, but rather have a way to detect and handle this situation in code 😅